<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceInfoDialog</class>
 <widget class="QDialog" name="DeviceInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>新建列</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Config Information 1</string>
      </property>
      <item>
       <property name="text">
        <string>SDT Info</string>
       </property>
       <item>
        <property name="text">
         <string>Model</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Version</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AxisNum</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ComType</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Current Device Info</string>
       </property>
       <item>
        <property name="text">
         <string>SoftVersion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FirmwareVersion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FirmwareDate</string>
        </property>
       </item>
      </item>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
